### PR TITLE
Homework1: Alexey Horongziy

### PR DESCRIPTION
**Pull request by AlSurick**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/random_number.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import random
+
```
Comment (belovmd) [inline src/random_number.py:3]: 1 line between import and code

File: src/prime_numbers.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import itertools
+
+
+def iter_primes():
```
Comment (belovmd) [inline src/prime_numbers.py:5]: 2 lines between import and code

File: src/triple_quoted_str.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+if __name__ == '__main__':
+    REFRAIN = '''
+    %d bottles of beer on the wall,
+    %d bottles of beer,
+    take one down, pass it around,
+    %d bottles of beer on the wall!
+    '''
+    bottles_of_beer = 9
```
Comment (belovmd) [inline src/triple_quoted_str.py:10]: no lines between constant and code

File: src/queens_problem_ex.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
+BOARD_SIZE = 8
```
Comment (belovmd) [inline src/queens_problem_ex.py:2]: 2 lines between constants and code

**Review by belovmd — CHANGES_REQUESTED** [review]
missed sanity checks - missed tasks description

File: src/queens_problem_ex.py (hunk 4 above)
Comment (AlSurick) [reply src/queens_problem_ex.py:2]: Sorry, but I don't understand what I must change

**Review by belovmd — APPROVED** [review]
1

**Pull request closed without merging**